mp/src/game/server/momentum/mom_triggers.cpp

```diff
@@ -1557,6 +1557,30 @@ void CNoGrenadesZone::Precache()
     PrecacheModel(NOGRENADE_SPRITE);
 }
 
+void CNoGrenadesZone::OnStartTouch(CBaseEntity* pOther)
+{
+    if (!pOther)
+        return;
+
+    const auto pSticky = dynamic_cast<CMomStickybomb *>(pOther);
+    if (!pSticky)
+        return;
+
+    pSticky->SetCanExplode(false);
+}
+
+void CNoGrenadesZone::OnEndTouch(CBaseEntity* pOther)
+{
+    if (!pOther)
+        return;
+
+    const auto pSticky = dynamic_cast<CMomStickybomb *>(pOther);
+    if (!pSticky)
+        return;
+
+    pSticky->SetCanExplode(true);
+}
```

Comment:
Always call your base class methods.

Reply:
Ah I glanced at these and figured I didn't have to as it was prefixed "On", woops